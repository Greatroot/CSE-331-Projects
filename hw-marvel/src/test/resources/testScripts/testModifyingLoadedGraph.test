# Tests that a loaded graph can be modified after the fact.

LoadGraph g testData.tsv

AddNode g A
AddNode g B

ListNodes g