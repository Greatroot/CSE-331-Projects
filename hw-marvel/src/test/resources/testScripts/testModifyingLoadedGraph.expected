# Tests that a loaded graph can be modified after the fact.

loaded graph g

added node A to g
added node B to g

g contains: A B n1 n10 n11 n12 n13 n2 n3 n4 n5 n6 n7 n8 n9